src/AppBundle/EventListener/ControllerListener.php

```diff
@@ -35,6 +38,9 @@ public function __construct(SourceCodeExtension $twigExtension)
         $this->twigExtension = $twigExtension;
     }
 
+    /**
+     * @param FilterControllerEvent $event
+     */
```

Comment:
This docblock doesn't add any value (the parameter type can be derived from the type hint).